Detectors/FIT/FV0/reconstruction/include/FV0Reconstruction/CTFCoder.h

```diff
@@ -119,17 +119,19 @@ void CTFCoder::decode(const CTF::base& ec, VDIG& digitVec, VCHAN& channelVec)
   DECODEFV0(cd.bcInc,     CTF::BLC_bcInc);
   DECODEFV0(cd.orbitInc,  CTF::BLC_orbitInc);
   DECODEFV0(cd.nChan,     CTF::BLC_nChan);
-
   DECODEFV0(cd.idChan,    CTF::BLC_idChan);
-  DECODEFV0(cd.time,      CTF::BLC_time);
-  DECODEFV0(cd.charge,    CTF::BLC_charge);
-
+  DECODEFV0(cd.cfdTime,   CTF::BLC_cfdTime);
+  DECODEFV0(cd.qtcAmpl,   CTF::BLC_qtcAmpl);
   // extra slot was added in the end
   DECODEFV0(cd.trigger,   CTF::BLC_trigger);
-  // triggers were added later, in old data they are absent:
+  DECODEFV0(cd.qtcChain,  CTF::BLC_qtcChain);
+  // triggers and qtcChain were added later, in old data they are absent:
   if (cd.trigger.empty()) {
     cd.trigger.resize(cd.header.nTriggers);
   }
+  if (cd.qtcChain.empty()) {
+    cd.qtcChain.resize(channelVec.size()); // TODO: is this correct? Or should I make it an additional CTFheader member?
```

Comment:
Hi, @shahor02 
Could you tell me if accessing `channelVec.size()` from her is correct?

Also, generally is the renaming proposed in this PR acceptable? (I haven't tested it with older CTFs yet. Proper testing it on my side may take up to Tuesday...)

Reply:
No, it is wrong, since the channelVec is filled only in https://github.com/AliceO2Group/AliceO2/blob/8390379d0a285430697565cd6c4b5372d4a5b1b0/Detectors/FIT/FV0/reconstruction/include/FV0Reconstruction/CTFCoder.h#L137 
But you can do e.g.
```
if (cd.qtcChain.empty()) {
    cd.qtcChain.resize(cd.cfdTime.size()); 
  }
```
to ensure correct qtcChain size for old data.
Concerning renamings: it is irrelevant, the CTF tree internally contains only names like `<DET>_BLOCK_<i>`

Reply:
OK, thanks for the advice.